packages/carbon-web-components/src/components/button/defs.ts

```diff
@@ -1,16 +1,16 @@
 /**
  * @license
  *
- * Copyright IBM Corp. 2020
+ * Copyright IBM Corp. 2020, 2023
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
 /**
- * Button kinds.
+ * Button type.
  */
-export enum BUTTON_KIND {
+export enum BUTTON_TYPE {
```

Comment:
Looks like ButtonGroup uses the `BUTTON_KIND` enum, should we update in buttongroup?

<img width="1187" alt="Screen Shot 2023-03-20 at 3 35 37 PM" src="https://user-images.githubusercontent.com/54281166/226460210-810096b8-4365-46e7-8acc-c730259f8b50.png">

Reply:
Whoops, I think I misread some docs somewhere and thought the attribute's name changed to `type`, but it actually remains the same lol. Will change back, thanks for making me notice :)